Slim/Player/ProtocolHandlers.pm

```diff
@@ -27,14 +27,13 @@ my %protocolHandlers = (
 	tmp      => qw(Slim::Player::Protocols::Volatile),
 	mms      => qw(Slim::Player::Protocols::MMS),
 	spdr     => qw(Slim::Player::Protocols::SqueezePlayDirect),
+	http     => qw(Slim::Player::Protocols::HTTP),
+	https    => qw(Slim::Player::Protocols::HTTPS),
+	icy      => qw(Slim::Player::Protocols::HTTP),	
 	playlist => 0,
 	db       => 1,
 );
 
-setHTTPHandler($prefs->get('useBufferedHTTP'));
-
-$prefs->setChange(sub { setHTTPHandler($_[1]) }, 'useBufferedHTTP');
```

Comment:
`$prefs` and `Slim::Utils::Prefs` are no longer being used in this module. Please remove.

Reply:
Done

Reply:
Done